Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPITestCocoa.mm

```diff
@@ -361,6 +373,74 @@ static void assertEquals(JSContextRef context, bool result, NSString *expectedSt
     return assertResolves(context, result, message);
 }
 
+JSValue *WebExtensionAPITest::addTest(JSContextRef context, JSValue *testFunction, NSString *testName)
```

Comment:
You can infer the test name from the function, which would be a good way to add tests with just one argument. 

```cpp
if (!testName.length)
    testName = testFunction[@"name"].toString;
```

`testName` would also be better as the first argument and optional.

```javascript
browser.test.addTest('runtime.getURL', async () => { })
```

So the function can flow inline after the test name (if not provided by the function name).

And this is still valid:

```javascript
browser.test.addTest(async runtimeGetURL() => { })
```